### ISSUE TITLE
Url with quotes is transformed to invalid CSS

### ISSUE THREAD

**Issue by cakebaker** [Compass/compass#125]
If I use quotes when specifying the url of a background image as shown below:

```
#openid_identifier {
  background: white url('/images/openid-16x16-transparent.png') no-repeat scroll 2pt 50%;
}
```

then Compass generates the following CSS code:

```
#openid_identifier { background: white url('"/images/openid-16x16-transparent.png"') no-repeat scroll 2pt 50%; }
```

Notice that the url is now enclosed by single and double quotes, which is not correct (and the browsers don't render the specified image). 

A workaround is to specify urls without any quotes. 

I'm using Compass 0.10.0.rc5

**Comment by nex3** [comment]
This is a Sass bug. I believe it's fixed in Sass 3.0.0.rc.4.

**Comment by chriseppstein** [comment]
cakebaker, can you verify that this is fixed?

**Comment by chriseppstein** [comment]
This was actually a compass bug. It'll be fixed in the next release.

**Comment by chriseppstein** [comment]
[Compass Core] Fix a bug that caused quotes strings passed to url() to be wrapped in another set of single quotes. Closed by 89442c93b19bd40d588770f8887b2644cf9f955d.
